pkg/gatewayserver/scheduling/scheduler.go

```diff
@@ -120,9 +120,9 @@ func NewScheduler(ctx context.Context, fps map[string]*frequencyplans.FrequencyP
 			if subBands := fp.SubBands; len(subBands) > 0 {
 				for _, subBand := range subBands {
 					params := SubBandParameters{
-						MinFrequency: subBand.MinFrequency,
-						MaxFrequency: subBand.MaxFrequency,
-						DutyCycle:    subBand.DutyCycle,
+						MinFrequency:   subBand.MinFrequency,
+						MaxFrequency:   subBand.MaxFrequency,
+						DutyCycleLimit: subBand.DutyCycle,
```

Comment:
Nit: I'm a bit pedantic here but I don't know if `DutyCycleLimit` is correct.

```
Duty cycle = On-time/total-time
```

Afaik, duty cycle already implies "max" on-time or "limit" of on-time. So I feel like the original  `sb.DutyCycle` was the correct usage.

Perhaps `AllowedDutyCycle` is technically more correct but this is not really important.